src/cli.rs

```diff
@@ -90,7 +90,7 @@ pub enum Command {
     UpdateWeakauras,
     /// Install an addon from the command line
     Install {
-        #[structopt(parse(try_from_str = str_to_flavor), possible_values = &["retail","ptr","beta","classic","classic_ptr"])]
+        #[structopt(parse(try_from_str = str_to_flavor), possible_values = &["retail","ptr","beta","classic","classic_ptr","classic_beta"])]
```

Comment:
@tarkah i was wondering if we could take this and move it to `config/wow` so we could have a method on Flavor which returned all we needed from there? Just so we don't miss anything when adding new flavors.